oxd-server/src/main/java/org/gluu/oxd/server/persistence/service/GluuPersistenceService.java

```diff
@@ -0,0 +1,317 @@
+package org.gluu.oxd.server.persistence.service;
+
+import org.gluu.oxd.common.ExpiredObject;
+import org.gluu.oxd.common.ExpiredObjectType;
+import org.gluu.oxd.common.Jackson2;
+import org.gluu.oxd.common.PersistenceConfigKeys;
+import org.gluu.oxd.server.OxdServerConfiguration;
+import org.gluu.oxd.server.persistence.modal.OrganizationBranch;
+import org.gluu.oxd.server.persistence.modal.RpObject;
+import org.gluu.oxd.server.persistence.providers.GluuPersistenceConfiguration;
+import org.gluu.oxd.server.persistence.providers.PersistenceEntryManagerFactory;
+import org.gluu.oxd.server.service.MigrationService;
+import org.gluu.oxd.server.service.Rp;
+import org.gluu.persist.PersistenceEntryManager;
+import org.gluu.persist.exception.EntryPersistenceException;
+import org.gluu.persist.model.base.SimpleBranch;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Properties;
+import java.util.Set;
+
+public class GluuPersistenceService implements PersistenceService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(GluuPersistenceService.class);
+    private OxdServerConfiguration configuration;
+    private PersistenceEntryManager persistenceEntryManager;
+    private String persistenceType;
+    private String BASE_DN = "o=gluu";
```

Comment:
Add configuration property for baseDn to oxd-server.yml.

Reply:
Done